tests/test_utils.py

```diff
@@ -5,27 +5,49 @@
     convert_to_list,
     convert_values_to_list,
     get_latest_sdk_version,
+    filter_exact_match,
 )
 
 
 class TestUtils(TestCase):
     def test_single_item_list(self):
-        assert convert_to_list("a") == ["a"]
+        self.assertEqual(convert_to_list("a"), ["a"])
 
     def test_multiple_items_list(self):
-        actual = convert_to_list(["a", "b"])
-        assert actual == ["a", "b"] or actual == ["b", "a"]
+        self.assertCountEqual(convert_to_list(["a", "b"]), ["a", "b"])
 
     def test_convert_values_to_list(self):
-        d = {1: None, 2: "b", 3: ["c"]}
+        actual = convert_values_to_list({1: None, 2: "b", 3: ["c"]})
 
         expected = {1: [], 2: ["b"], 3: ["c"]}
 
-        assert convert_values_to_list(d) == expected
+        assert actual == expected
 
     def test_get_latest_sdk_version(self):
         sdk_old_version = "0.18.0"
 
         lv = get_latest_sdk_version()
 
         assert lv > LooseVersion(sdk_old_version)
+
+    def test_filter_exact_match_single_name(self):
+        search_result = [{"name": "China"}, {"name": "South China"}]
+
+        actual = filter_exact_match("China", search_result)
+
+        expected = [{"name": "China"}]
+
+        self.assertEqual(actual, expected)
+
```

Comment:
We also need a test for filter_exact_match correctly returning an empty set (e.g. searching for "Chi")